src/components/Accordion.js

```diff
@@ -40,6 +41,10 @@ const Accordion = React.createClass({
       item.onOpened();
     }
 
+    if (item.pref) {
+      prefs[item.pref] = opened[i];
+    }
```

Comment:
hmm, I think `pref` might make more sense outside of Accordion.

So, in this case we could add this to an `onOpened` callback